<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="9" failed="2" total="18" passed="7">
  <reporter-output>
  </reporter-output>
  <suite name="TestSuite" duration-ms="42836" started-at="2017-12-04T16:23:52Z" finished-at="2017-12-04T16:24:35Z">
    <groups>
    </groups>
    <test name="InternetExplorerTest" duration-ms="23300" started-at="2017-12-04T16:23:52Z" finished-at="2017-12-04T16:24:16Z">
      <class name="Homework4.Homework4.MainTests">
        <test-method status="PASS" signature="setUp(java.lang.String)[pri:0, instance:Homework4.Homework4.MainTests@bbe345]" name="setUp" is-config="true" duration-ms="6081" started-at="2017-12-04T18:23:52Z" finished-at="2017-12-04T18:23:59Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[InternetExplorer]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="logIn(java.lang.String, java.lang.String)[pri:0, instance:Homework4.Homework4.MainTests@bbe345]" name="logIn" duration-ms="17084" started-at="2017-12-04T18:23:59Z" data-provider="authorization" finished-at="2017-12-04T18:24:16Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[webinar.test@gmail.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Xcg7299bnSmMuRLp9ITw]]>
              </value>
            </param>
          </params>
          <exception class="org.openqa.selenium.NoSuchElementException">
            <message>
              <![CDATA[Unable to find element with css selector == #email
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.5.3', revision: 'a88d25fe6b', time: '2017-08-29T12:42:44.417Z'
System info: host: 'ROMAN-PC', ip: '192.168.1.4', os.name: 'Windows 7', os.arch: 'x86', os.version: '6.1', java.version: '1.8.0_151'
Driver info: org.openqa.selenium.ie.InternetExplorerDriver
Capabilities [{acceptInsecureCerts=false, browserVersion=11, se:ieOptions={nativeEvents=true, browserAttachTimeout=0, ie.ensureCleanSession=false, elementScrollBehavior=0, enablePersistentHover=true, ie.browserCommandLineSwitches=, ie.forceCreateProcessApi=false, requireWindowFocus=false, initialBrowserUrl=http://localhost:48093/, ignoreZoomSetting=false, ie.fileUploadDialogTimeout=3000, ignoreProtectedModeSettings=false}, browserName=internet explorer, pageLoadStrategy=normal, javascriptEnabled=true, platformName=XP, setWindowRect=true, platform=XP}]
Session ID: efe6c63e-f7a9-47e3-9562-3a47336ef4fd
*** Element info: {Using=id, value=email}]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.NoSuchElementException: Unable to find element with css selector == #email
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.5.3', revision: 'a88d25fe6b', time: '2017-08-29T12:42:44.417Z'
System info: host: 'ROMAN-PC', ip: '192.168.1.4', os.name: 'Windows 7', os.arch: 'x86', os.version: '6.1', java.version: '1.8.0_151'
Driver info: org.openqa.selenium.ie.InternetExplorerDriver
Capabilities [{acceptInsecureCerts=false, browserVersion=11, se:ieOptions={nativeEvents=true, browserAttachTimeout=0, ie.ensureCleanSession=false, elementScrollBehavior=0, enablePersistentHover=true, ie.browserCommandLineSwitches=, ie.forceCreateProcessApi=false, requireWindowFocus=false, initialBrowserUrl=http://localhost:48093/, ignoreZoomSetting=false, ie.fileUploadDialogTimeout=3000, ignoreProtectedModeSettings=false}, browserName=internet explorer, pageLoadStrategy=normal, javascriptEnabled=true, platformName=XP, setWindowRect=true, platform=XP}]
Session ID: efe6c63e-f7a9-47e3-9562-3a47336ef4fd
*** Element info: {Using=id, value=email}
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at org.openqa.selenium.remote.http.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:185)
	at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:120)
	at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:49)
	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:164)
	at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:82)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:646)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:416)
	at org.openqa.selenium.remote.RemoteWebDriver.findElementById(RemoteWebDriver.java:462)
	at org.openqa.selenium.By$ById.findElement(By.java:218)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:408)
	at Homework4.Homework4.MainTests.logIn(MainTests.java:53)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:400)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.NoSuchElementException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logIn -->
        <test-method status="SKIP" signature="openGoodsPage()[pri:0, instance:Homework4.Homework4.MainTests@bbe345]" name="openGoodsPage" duration-ms="1" started-at="2017-12-04T18:24:16Z" depends-on-methods="Homework4.Homework4.MainTests.logIn" finished-at="2017-12-04T18:24:16Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method MainTests.openGoodsPage()[pri:0, instance:Homework4.Homework4.MainTests@bbe345] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method MainTests.openGoodsPage()[pri:0, instance:Homework4.Homework4.MainTests@bbe345] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1037)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:400)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openGoodsPage -->
        <test-method status="SKIP" signature="activateNewGood()[pri:0, instance:Homework4.Homework4.MainTests@bbe345]" name="activateNewGood" duration-ms="0" started-at="2017-12-04T18:24:16Z" depends-on-methods="Homework4.Homework4.MainTests.openGoodsPage" finished-at="2017-12-04T18:24:16Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method MainTests.activateNewGood()[pri:0, instance:Homework4.Homework4.MainTests@bbe345] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method MainTests.activateNewGood()[pri:0, instance:Homework4.Homework4.MainTests@bbe345] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1037)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:400)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- activateNewGood -->
        <test-method status="SKIP" signature="saveGood()[pri:0, instance:Homework4.Homework4.MainTests@bbe345]" name="saveGood" duration-ms="0" started-at="2017-12-04T18:24:16Z" depends-on-methods="Homework4.Homework4.MainTests.activateNewGood" finished-at="2017-12-04T18:24:16Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method MainTests.saveGood()[pri:0, instance:Homework4.Homework4.MainTests@bbe345] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method MainTests.saveGood()[pri:0, instance:Homework4.Homework4.MainTests@bbe345] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1037)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:400)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- saveGood -->
        <test-method status="SKIP" signature="openShop()[pri:0, instance:Homework4.Homework4.MainTests@bbe345]" name="openShop" duration-ms="0" started-at="2017-12-04T18:24:16Z" depends-on-methods="Homework4.Homework4.MainTests.saveGood" finished-at="2017-12-04T18:24:16Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method MainTests.openShop()[pri:0, instance:Homework4.Homework4.MainTests@bbe345] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method MainTests.openShop()[pri:0, instance:Homework4.Homework4.MainTests@bbe345] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1037)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:400)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openShop -->
        <test-method status="SKIP" signature="checkGood()[pri:0, instance:Homework4.Homework4.MainTests@bbe345]" name="checkGood" duration-ms="0" started-at="2017-12-04T18:24:16Z" depends-on-methods="Homework4.Homework4.MainTests.openShop" finished-at="2017-12-04T18:24:16Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method MainTests.checkGood()[pri:0, instance:Homework4.Homework4.MainTests@bbe345] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method MainTests.checkGood()[pri:0, instance:Homework4.Homework4.MainTests@bbe345] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1037)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:400)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkGood -->
      </class> <!-- Homework4.Homework4.MainTests -->
    </test> <!-- InternetExplorerTest -->
    <test name="FirefoxTest" duration-ms="38351" started-at="2017-12-04T16:23:52Z" finished-at="2017-12-04T16:24:31Z">
      <class name="Homework4.Homework4.MainTests">
        <test-method status="PASS" signature="setUp(java.lang.String)[pri:0, instance:Homework4.Homework4.MainTests@1be0360]" name="setUp" is-config="true" duration-ms="9402" started-at="2017-12-04T18:23:52Z" finished-at="2017-12-04T18:24:02Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Firefox]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="logIn(java.lang.String, java.lang.String)[pri:0, instance:Homework4.Homework4.MainTests@1be0360]" name="logIn" duration-ms="15021" started-at="2017-12-04T18:24:02Z" data-provider="authorization" finished-at="2017-12-04T18:24:17Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[webinar.test@gmail.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Xcg7299bnSmMuRLp9ITw]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logIn -->
        <test-method status="FAIL" signature="openGoodsPage()[pri:0, instance:Homework4.Homework4.MainTests@1be0360]" name="openGoodsPage" duration-ms="13896" started-at="2017-12-04T18:24:17Z" depends-on-methods="Homework4.Homework4.MainTests.logIn" finished-at="2017-12-04T18:24:31Z">
          <exception class="org.openqa.selenium.TimeoutException">
            <message>
              <![CDATA[Expected condition failed: waiting for visibility of element located by By.xpath: //*[@data-submenu='10'] (tried for 10 second(s) with 500 MILLISECONDS interval)
Build info: version: '3.5.3', revision: 'a88d25fe6b', time: '2017-08-29T12:42:44.417Z'
System info: host: 'ROMAN-PC', ip: '192.168.1.4', os.name: 'Windows 7', os.arch: 'x86', os.version: '6.1', java.version: '1.8.0_151'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{moz:profile=C:\TEMP\rust_mozprofile.KD5Df2DCD6pu, rotatable=false, timeouts={implicit=0, page load=300000, script=30000}, pageLoadStrategy=normal, platform=XP, specificationLevel=0, moz:accessibilityChecks=false, acceptInsecureCerts=false, browserVersion=52.0.1, platformVersion=6.1, moz:processID=11148, browserName=firefox, javascriptEnabled=true, platformName=XP}]
Session ID: 1f6b3e29-f7f8-4fb0-825e-c91a38814054]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.TimeoutException: Expected condition failed: waiting for visibility of element located by By.xpath: //*[@data-submenu='10'] (tried for 10 second(s) with 500 MILLISECONDS interval)
Build info: version: '3.5.3', revision: 'a88d25fe6b', time: '2017-08-29T12:42:44.417Z'
System info: host: 'ROMAN-PC', ip: '192.168.1.4', os.name: 'Windows 7', os.arch: 'x86', os.version: '6.1', java.version: '1.8.0_151'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{moz:profile=C:\TEMP\rust_mozprofile.KD5Df2DCD6pu, rotatable=false, timeouts={implicit=0, page load=300000, script=30000}, pageLoadStrategy=normal, platform=XP, specificationLevel=0, moz:accessibilityChecks=false, acceptInsecureCerts=false, browserVersion=52.0.1, platformVersion=6.1, moz:processID=11148, browserName=firefox, javascriptEnabled=true, platformName=XP}]
Session ID: 1f6b3e29-f7f8-4fb0-825e-c91a38814054
	at org.openqa.selenium.support.ui.WebDriverWait.timeoutException(WebDriverWait.java:80)
	at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:232)
	at Homework4.Homework4.MainTests.openGoodsPage(MainTests.java:68)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:400)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.TimeoutException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openGoodsPage -->
        <test-method status="SKIP" signature="activateNewGood()[pri:0, instance:Homework4.Homework4.MainTests@1be0360]" name="activateNewGood" duration-ms="0" started-at="2017-12-04T18:24:31Z" depends-on-methods="Homework4.Homework4.MainTests.openGoodsPage" finished-at="2017-12-04T18:24:31Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method MainTests.activateNewGood()[pri:0, instance:Homework4.Homework4.MainTests@1be0360] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method MainTests.activateNewGood()[pri:0, instance:Homework4.Homework4.MainTests@1be0360] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1037)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:400)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- activateNewGood -->
        <test-method status="SKIP" signature="saveGood()[pri:0, instance:Homework4.Homework4.MainTests@1be0360]" name="saveGood" duration-ms="0" started-at="2017-12-04T18:24:31Z" depends-on-methods="Homework4.Homework4.MainTests.activateNewGood" finished-at="2017-12-04T18:24:31Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method MainTests.saveGood()[pri:0, instance:Homework4.Homework4.MainTests@1be0360] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method MainTests.saveGood()[pri:0, instance:Homework4.Homework4.MainTests@1be0360] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1037)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:400)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- saveGood -->
        <test-method status="SKIP" signature="openShop()[pri:0, instance:Homework4.Homework4.MainTests@1be0360]" name="openShop" duration-ms="0" started-at="2017-12-04T18:24:31Z" depends-on-methods="Homework4.Homework4.MainTests.saveGood" finished-at="2017-12-04T18:24:31Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method MainTests.openShop()[pri:0, instance:Homework4.Homework4.MainTests@1be0360] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method MainTests.openShop()[pri:0, instance:Homework4.Homework4.MainTests@1be0360] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1037)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:400)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openShop -->
        <test-method status="SKIP" signature="checkGood()[pri:0, instance:Homework4.Homework4.MainTests@1be0360]" name="checkGood" duration-ms="0" started-at="2017-12-04T18:24:31Z" depends-on-methods="Homework4.Homework4.MainTests.openShop" finished-at="2017-12-04T18:24:31Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method MainTests.checkGood()[pri:0, instance:Homework4.Homework4.MainTests@1be0360] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method MainTests.checkGood()[pri:0, instance:Homework4.Homework4.MainTests@1be0360] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1037)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:400)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkGood -->
      </class> <!-- Homework4.Homework4.MainTests -->
    </test> <!-- FirefoxTest -->
    <test name="ChromeTest" duration-ms="42836" started-at="2017-12-04T16:23:52Z" finished-at="2017-12-04T16:24:35Z">
      <class name="Homework4.Homework4.MainTests">
        <test-method status="PASS" signature="setUp(java.lang.String)[pri:0, instance:Homework4.Homework4.MainTests@1f8bee4]" name="setUp" is-config="true" duration-ms="3555" started-at="2017-12-04T18:23:52Z" finished-at="2017-12-04T18:23:56Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="logIn(java.lang.String, java.lang.String)[pri:0, instance:Homework4.Homework4.MainTests@1f8bee4]" name="logIn" duration-ms="14455" started-at="2017-12-04T18:23:56Z" data-provider="authorization" finished-at="2017-12-04T18:24:10Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[webinar.test@gmail.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Xcg7299bnSmMuRLp9ITw]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logIn -->
        <test-method status="PASS" signature="openGoodsPage()[pri:0, instance:Homework4.Homework4.MainTests@1f8bee4]" name="openGoodsPage" duration-ms="10219" started-at="2017-12-04T18:24:10Z" depends-on-methods="Homework4.Homework4.MainTests.logIn" finished-at="2017-12-04T18:24:21Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openGoodsPage -->
        <test-method status="PASS" signature="activateNewGood()[pri:0, instance:Homework4.Homework4.MainTests@1f8bee4]" name="activateNewGood" duration-ms="7858" started-at="2017-12-04T18:24:21Z" depends-on-methods="Homework4.Homework4.MainTests.openGoodsPage" finished-at="2017-12-04T18:24:29Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- activateNewGood -->
        <test-method status="PASS" signature="saveGood()[pri:0, instance:Homework4.Homework4.MainTests@1f8bee4]" name="saveGood" duration-ms="685" started-at="2017-12-04T18:24:29Z" depends-on-methods="Homework4.Homework4.MainTests.activateNewGood" finished-at="2017-12-04T18:24:29Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- saveGood -->
        <test-method status="PASS" signature="openShop()[pri:0, instance:Homework4.Homework4.MainTests@1f8bee4]" name="openShop" duration-ms="2675" started-at="2017-12-04T18:24:29Z" depends-on-methods="Homework4.Homework4.MainTests.saveGood" finished-at="2017-12-04T18:24:32Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openShop -->
        <test-method status="PASS" signature="checkGood()[pri:0, instance:Homework4.Homework4.MainTests@1f8bee4]" name="checkGood" duration-ms="3335" started-at="2017-12-04T18:24:32Z" depends-on-methods="Homework4.Homework4.MainTests.openShop" finished-at="2017-12-04T18:24:35Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkGood -->
      </class> <!-- Homework4.Homework4.MainTests -->
    </test> <!-- ChromeTest -->
  </suite> <!-- TestSuite -->
</testng-results>
